examples/00_quick_start/nn_embedding_ranker_movielens.ipynb

```diff
@@ -0,0 +1,10 @@
+{
```

Comment:
not sure if there was an error in the submission of the notebook, but I can't see it